client_test.go

```diff
@@ -228,7 +357,7 @@ func testResume(t *testing.T, expired, preimageRevealed, expectSuccess bool) {
 	signalSwapPaymentResult := ctx.AssertPaid(swapInvoiceDesc)
 	signalPrepaymentResult := ctx.AssertPaid(prepayInvoiceDesc)
 
-	// Expect client to register for conf
+	// Expect client to register for conf.
```

Comment:
Nit: comment fixes can be separate commit. But only if you want to be a purist :)

Reply:
Sure, purist sounds good :D